### PR TITLE
[SF7] Add Symfony 7 support for Customer bundle

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
|-----------------|-----
| Branch?         | symfony-7
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

After
image

### CODE REVIEW

File: src/Sylius/Bundle/CustomerBundle/.gitignore [hunk 1]
```diff
@@ -6,3 +6,7 @@ composer.lock
 
 test/app/cache
 test/app/logs
+
```
Comment (diimpp) [inline src/Sylius/Bundle/CustomerBundle/.gitignore:11]: Please format it the same as for other bundles.

image

File: src/Sylius/Bundle/AttributeBundle/.phpunit.result.cache [hunk 2]
```diff
@@ -0,0 +1 @@
+{"version":1,"defects":{"Sylius\\Bundle\\AttributeBundle\\Tests\\SyliusAttributeBundleTest::its_services_are_initializable":4},"times":{"Sylius\\Bundle\\AttributeBundle\\Tests\\SyliusAttributeBundleTest::its_services_are_initializable":0.039,"Sylius\\Bundle\\AttributeBundle\\Tests\\DependencyInjection\\SyliusAttributeExtensionTest::it_autoconfigures_attribute_type_with_attribute":0.016,"Sylius\\Bundle\\AttributeBundle\\Tests\\Form\\Type\\AttributeType\\SelectAttributeTypeTest::it_return_all_choices":0.015}}
```
Comment (diimpp) [inline src/Sylius/Bundle/AttributeBundle/.phpunit.result.cache]: .gitignore

Comment (loic425) [reply src/Sylius/Bundle/AttributeBundle/.phpunit.result.cache]: yeah, wrong commit amend

File: src/Sylius/Bundle/CustomerBundle/.gitignore (hunk 1 above)
Comment (loic425) [reply src/Sylius/Bundle/CustomerBundle/.gitignore:11]: fixed

**Review by diimpp — APPROVED** [review]

**Pull request merged**